themes/index.js

```diff
@@ -462,6 +462,11 @@ export const themes = {
     icon_color: "ffffff",
     bg_color: "35,4158d0,c850c0,ffcc70",
   },
+  lucario: {
+    title_color: "4aa5d4",
+    text_color: "fffd7d",
+    icon_color: "da6771",
+    bg_color: "1b2333",
 };
```

Comment:
It looks like you started adding a new theme variant named **lucario**, but it seems that you opened the `{` for this new theme but forgot to close it. Could you please close the `{` to ensure the new theme is properly defined?

```suggestion
  lucario: {
    title_color: "4aa5d4",
    text_color: "fffd7d",
    icon_color: "da6771",
    bg_color: "1b2333",
    },  
};
```